storage-market.md

```diff
@@ -119,6 +119,27 @@ The `power table` is exported by the storage market for use by consensus. There
 
 TODO: rephrase the above to make it clear that power is updated only on PoSt submission and when slashed.
 
+To check the power of a given miner, use the following:
+
+```go
+func GetMinersPowerAt(ts TipSet, m Address) Integer {
```

Comment:
can you clarify in which situations this would be actually called?

Reply:
argh. I could have used this working on #403. Hard to read through PRs before starting one. :(